components/display/index.rst

```diff
@@ -482,6 +482,43 @@ And then later in code:
     - Axis labels are currently not possible without manually placing them.
     - The grid and border color is set with it.graph(), while the traces are defined separately.
 
+QR Codes
+********
+
+Use this component to generate a qr code containing a string on the device, you can then draw the qr code on compatible displays.
```

Comment:
```suggestion
Use this component to generate a QR-code containing a string on the device, which can then be drawn on compatible displays.
```

It might be nice to explain which displays are compatible as well.